run/start.sh

```diff
@@ -5,10 +5,18 @@ if [ ! -f /hath/data/client_login ]; then
 	printf "${HATH_CLIENT_ID}-${HATH_CLIENT_KEY}" >> /hath/data/client_login
 fi
 
-exec java -jar /opt/hath/HentaiAtHome.jar \
-    --cache-dir=/hath/cache               \
-    --data-dir=/hath/data                 \
-    --download-dir=/hath/download         \
-    --log-dir=/hath/log                   \
-    --temp-dir=/hath/tmp
-
+if [ "$IPV4"=true ]; then
```

Comment:
Instead of duplicating this here, maybe the next line can be

```
exec java -Djava.net.preferIPv4Stack=$PREFER_IPV4_STACK -jar /opt/hath/HentaiAtHome.jar \
```

And the `Dockerfile` can be modified to have `ENV PREFER_IPV4_STACK=false`